File: vscode-extension/src/protocol/AssemblyData.ts
/*------------------------------------------------------------------------------------------------
 *  Copyright (c) 2021 ICSharpCode
 *  Licensed under the MIT License. See LICENSE.TXT in the project root for license information.
 *-----------------------------------------------------------------------------------------------*/

export default interface AssemblyData {
  name: string;
  filePath: string;
  version?: string;
  targetFramework?: string;
}

File: backend/src/ILSpy.Backend/Model/AssemblyData.cs
// Copyright (c) 2021 ICSharpCode
// Licensed under the MIT license. See the LICENSE file in the project root for more information.

namespace ILSpy.Backend.Model
{
    public record AssemblyData(
        string Name,
        string FilePath)
    {
        public string? Version { get; set; }
        public string? TargetFramework { get; set; }
    }
}
